primitives/account/src/accounts.rs

```diff
@@ -541,7 +609,7 @@ impl Accounts {
         )
     }
 
-    /// Merges the log transactions from senders, receiptients and create_contracts and returns batch_info containing the merged tx_logs.
+    /// Merges the log transactions from senders, recipients and create_contracts and returns batch_info containing the merged tx_logs.
     /// This means that the batch_info inherent_logs and receipts are preserved and returned by this method.
     /// The batch_info provided as argument is expected to have its tx_logs vec empty, otherwise its content will get overwritten.
```

Comment:
Update the batch_info comment line explaining the need for a empty tx_log. Something like:
For macroblocks the vec should be empty, for micro it should contain the payout rewards. Thus if there are tx_logs in batch_info, there will be no changes from this function. Otherwise, the merged tx_logs will be returned.